src/MessageHandler/ZipDatasetFilesHandler.php

```diff
@@ -91,27 +91,27 @@ public function __construct(
      */
     public function __invoke(ZipDatasetFiles $zipDatasetFiles)
     {
-        $this->logger->info('Zipping files started');
```

Comment:
Why was this moved? The process started. Maybe add this logger back, but say something like "zip worker started" or something.